db/migrate/20151008114043_create_chargeback_rate_detail_measure.rb

```diff
@@ -0,0 +1,12 @@
+class CreateChargebackRateDetailMeasure < ActiveRecord::Migration
+  def change
+    create_table :chargeback_rate_detail_measures do |t|
+      t.string :name
+      t.string :units, :array => true
```

Comment:
We've never used array types before, so this is new.  Not sure what the impact of this might be.  cc @gtanzillo @matthewd @blomquisg